src/webvr-polyfill.js

```diff
@@ -54,6 +54,11 @@ WebVRPolyfill.prototype.isDeprecatedWebVRAvailable = function() {
   return ('getVRDevices' in navigator) || ('mozGetVRDevices' in navigator);
 };
 
+WebVRPolyfill.prototype.connectDisplay = function(vrDisplay) {
+  vrDisplay.fireVRDisplayConnect_();
```

Comment:
FYI: last time I checked, Firefox, Chrome, and Chromium don't fire `vrdisplayconnect` when the page loads. only when a device is manually plugged in/reconected. that still make sense in this case?